<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>799</width>
       <height>559</height>
      </rect>
     </property>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell Extra Bold</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/mk.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_llegada">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS PGothic</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tasa de llegada</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_servicio">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>90</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS PGothic</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tasa de servicio</string>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_numeroServicio">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS PGothic</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> Número de servidores</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS PGothic</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/n.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="lbl_opciones">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS PGothic</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Opciones para cálculo de Pn:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_calcular">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell Extra Bold</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 21, 255);</string>
       </property>
       <property name="text">
        <string>Calcular</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/calculating.svg</normaloff>icons/calculating.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_exactamente">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>380</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Exactamente</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_alMenos">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Al menos</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_maximo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Máximo</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_sistema">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>380</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Sistema</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_cola">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>400</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Cola</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_llegada">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_servicio">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_numeroServicio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_cliente">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>321</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>280</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS PGothic</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string> Clientes</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>21</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS PGothic</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/k.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/lambda.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>90</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>icons/mu.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_limpiar">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>510</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell Extra Bold</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 21, 255);</string>
       </property>
       <property name="text">
        <string> Limpiar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/cleaning.svg</normaloff>icons/cleaning.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell Extra Bold</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>M/M/K</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>409</x>
        <y>0</y>
        <width>391</width>
        <height>561</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="lbl_solucion">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell Extra Bold</family>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Solución</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_P">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_P0">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>90</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Wq">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>180</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Lq">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Ws">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>270</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_Ls">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
